storage/src/vespa/storage/storageserver/mergethrottler.cpp

```diff
@@ -415,6 +416,7 @@ MergeThrottler::enqueueMerge(
     if (!validateNewMerge(mergeCmd, nodeSeq, msgGuard)) {
         return;
     }
+    _metrics->queueSize.addValue(_queue.size());
```

Comment:
Since queue size is conceptually a gauge-style metric, it feels more intuitive to me if we call `set()` instead of `addValue()`. The former is just an alias for the latter, so the end result should be the same

Reply:
K, fixed